sklearn/metrics/_base.py

```diff
@@ -200,3 +200,52 @@ def _average_multiclass_ovo_score(binary_metric, y_true, y_score,
         pair_scores[ix] = (a_true_score + b_true_score) / 2
 
     return np.average(pair_scores, weights=prevalence)
+
+
+def _check_ambiguity_pos_label(pos_label, y_true):
```

Comment:
This would better be renamed `_check_pos_label_consistency` or just `_check_pos_label`.